backend/api/models.py

```diff
@@ -2,4 +2,48 @@
 
 from django.db import models
 
-# Create your models here.
+class Business(models.Model):
+    camis = models.CharField(max_length=255, primary_key=True)
+    name = models.CharField(max_length=255)
+    address = models.CharField(max_length=255)
+    phone = models.CharField(max_length=10)
```

Comment:
you might want more than 10 characters for parentheses, dashes, spaces, country codes etc. Maybe 32 chars?